### PR TITLE
feat(tt): spruce up youtube route

### PR DESCRIPTION
**Pull request by vojtaholik**
screenshot

gif

### CODE REVIEW

File: apps/total-typescript/src/pages/youtube/[id].tsx [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 
   return (
-    <Layout footer={null} meta={{title: 'YouTube on Total TypeScript'}}>
-      <div className="video-responsive mt-16 sm:mt-40">
+    <Layout
+      footer={null}
+      meta={{
+        title: 'YouTube on Total TypeScript',
+        ogImage: {
+          url: `https://img.youtube.com/vi/${query.id}/maxresdefault.jpg`,
```
Comment (vojtaholik) [inline apps/total-typescript/src/pages/youtube/[id].tsx:17]: using video thumbnail for share card

File: apps/total-typescript/src/pages/youtube/[id].tsx [hunk 2]
```diff
@@ -20,8 +29,47 @@ const YouTube = () => {
...
 }
 
 export default YouTube
+
+const Banner = () => {
+  return (
+    <div className="sm:my-5 sm:p-5">
+      <div className="relative z-10 mx-auto flex w-full max-w-3xl flex-col items-center justify-around border border-gray-800 bg-gray-800/50 text-center text-gray-200 shadow-xl shadow-black/20 transition sm:flex-row sm:rounded-lg sm:text-left">
+        <Image
+          src={require('../../../public/assets/ts-book.png')}
```
Comment (joelhooks) [inline apps/total-typescript/src/pages/youtube/[id].tsx:44]: why this over cloudinary?

Comment (vojtaholik) [reply apps/total-typescript/src/pages/youtube/[id].tsx:44]: I just like the blurry placeholder but don't feel too strong about it

Comment (vojtaholik) [reply apps/total-typescript/src/pages/youtube/[id].tsx]: it's using cloudinary now

**Review by joelhooks — APPROVED** [review]

**Pull request merged**